docs/setting-up/client/postgresql.md

```diff
@@ -16,13 +16,9 @@ To monitor PostgreSQL queries, you must install a database extension. There are
 
 !!! alert alert-info "Note"
 
-    - `pg_stat_monitor` is the recommended option.
+    - Although `pg_stat_monitor` is beta software and currently unsupported, we do recommend the early adoption of `pg_stat_monitor`.
```

Comment:
We can avoid the conflict by not recommending any, but say what the status of each is:
pg_stat_monitor: new, beta, unsupported
pg_stat_statements: old, standard, supported